modules/vf-graphql-holochain/queries/action.ts

```diff
@@ -5,24 +5,24 @@
  * @since:   2019-12-23
  */
 
-import { DNAIdMappings } from '../types'
+import { DNAIdMappings, ReadParams } from '../types'
 import { mapZomeFn } from '../connection'
 
 import {
   Action,
 } from '@valueflows/vf-graphql'
 
 export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
-  const read = mapZomeFn(dnaConfig, conductorUri, 'specification', 'action', 'get_action')
-  const readAll = mapZomeFn(dnaConfig, conductorUri, 'specification', 'action', 'get_all_actions')
+  const read = mapZomeFn<ReadParams, Action>(dnaConfig, conductorUri, 'specification', 'action', 'get_action')
+  // const readAll = mapZomeFn<null, Action[]>(dnaConfig, conductorUri, 'specification', 'action', 'get_all_actions')
```

Comment:
oh sorry, this one is an exception to the others, and exists. plz uncomment. It is different because it doesn't use semantic_index, it just returns a flat pre-defined list that's defined in the code, as opposed to querying holochain